src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -17907,6 +17907,66 @@ BC42024: Unused local variable: 'x6'.
 
         End Sub
 
+        <Fact>
+        <WorkItem(15198, "https://github.com/dotnet/roslyn/issues/15198")>
+        Public Sub TuplePropertyArgs001()
```

Comment:
>TuplePropertyArgs001 [](start = 19, length = 20)

All tests here are for properties. Is this the only possible source of problems?